packages/odyssey-eslint/src/index.js

```diff
@@ -56,6 +57,7 @@ module.exports = {
       extends: [
         "plugin:jsx-a11y/recommended",
         "plugin:react-hooks/recommended",
+        "prettier",
```

Comment:
my understanding of eslint configuration extends and overrides is that this line is not necessary. These same file extensions are already targeted to use prettier in your changes above to other override objects

Reply:
has been removed!

Reply:
Looks like this was not removed?

Reply:
ah my mistake, I brought it back in to test something and forgot to remove it the second time. Has been removed for real this time!